core/src/main/java/me/aurium/beetle/api/util/Tuple2.java

```diff
@@ -1,4 +1,45 @@
 package me.aurium.beetle.api.util;
 
-public class Tuple2 {
+import java.util.Objects;
+
+public class Tuple2<A,B> {
```

Comment:
I highly discourage using anonymous tuples. Records were introduced for this reason. Ask yourself which is more readable:
```java
Tuple2<String, String> person;
String firstName = person.getA();
String lastName = person.getB();

Person person;
String firstName = person.firstName();
String lastName = person.lastName();
```

Reply:
eh

Reply:
what about no jdk 16 tho

Reply:
What about it? If someone wants to use beetle with a JDK version lower than has records, they can make their own tuples and let the rest of us use records

Reply:
Okay, probably a good idea then to remove the tuples